languages.yaml

```
@@ -79,6 +79,25 @@ Ada:
       - "#0018C9"
       - "#0C0A7C"
     chip: "#02F88C"
+Arduino:
+  type: programming
+  ascii: |
+    {0}    .--=====-:          :--=====-:    
+    {0}  :=====--=====-:    .-=====--=====-  
+    {0} ====:      .-====:.====-.      .====.
+    {0}-===           :======-.   :=:    -==-
+    {0}===-   =====.   .====:    -===-   :===
+    {0}:===           -======-.   .=.    ===-
+    {0} -===:.     .-====..-===-:      :==== 
+    {0}  :============-.    .-============:  
+    {0}     .--=====-:         .--=====-:               
+  colors:
+    ansi:
+      - cyan
+    hex:
+      - '#189BA1'
+    chip:
+      - "#F34B7D"
```

```suggestion
    chip: "#F34B7D"
```
This value isn't an array.